### PR TITLE
Adding a couple of Spanish franchises, removing generics, adding wikidata

### PR DESCRIPTION
**Pull request by ArenaL5**
As usual, this pull is an assortment of several unrelated commits.

The first three have a generic name added by mistake, which is finally removed at commit 85085ac2c73280d990aea7f3895993d1c3137132. The following commits should be pretty self-explanatory.

### CODE REVIEW

File: brands/amenity/ice_cream.json [hunk 1]
```diff
@@ -344,6 +344,34 @@
...
+      "fr",
+      "gn",
+      "it",
+      "ma",
+      "pa",
+      "pe",
+      "pt",
+      "re",
+      "sa",
+      "uk"
```
Comment (doublah) [inline brands/amenity/ice_cream.json]: Should this be gb country code for the UK?

Comment (ArenaL5) [reply brands/amenity/ice_cream.json]: You're right, it's my mistake. Why is United Kingdoms's code GB if they're going to reserve UK anyways? It's confusing.

Fixed with commit 75d1606ea19d0c7b3964743ff8bd5e1433b7f115. I also fixed it in other entries; some of them were opened by me, others weren't.

**Comment by ArenaL5** [comment]
I added another commit; this one's related to an old issue report (fe0e79ae164c97d6f903f6a8ebf5e1201c437d14).

**Comment by Adamant36** [comment]
@doublah, do you see anything that needs to fixed or altered with this? If not, it should probably be merged before @ArenaL5 adds anything else and it becomes even more ungainly to review. @ArenaL5, you really should do smaller PR's. There's zero reason to do ones with edits to 23 files and involving multiple open issues. Its semi-absurd.

**Comment by ArenaL5** [comment]
@Adamant36, you're right, I'm sorry. I won't add any other commit to this PR unless any of you want me to fix something.

I'll keep it in mind for future PRs.

**Comment by doublah** [comment]
LGTM

**Comment by Adamant36** [comment]
@arenaL5, no worries. Thanks for keeping it in mind in the future.

**Pull request merged**